Bottom: f49bb64048af03cff04b336861bfc0c3e1c0ec53
Top:    edd1b783d859c86749c1de597e03df81b0919495
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-03-30 03:15:30 -0300

Refresh of name-builtin-types.patch

---

diff --git a/include/gcc-cp-fe.def b/include/gcc-cp-fe.def
index 5137815..141ceef 100644
--- a/include/gcc-cp-fe.def
+++ b/include/gcc-cp-fe.def
@@ -531,7 +531,7 @@ GCC_METHOD3 (gcc_type, int_type,
 
 GCC_METHOD0 (gcc_type, char_type)
 
-/* Return a floating point type with the given properties.    If BUILTIN_NAME
+/* Return a floating point type with the given properties.  If BUILTIN_NAME
    is non-NULL, it must name a builtin integral type with the given
    signedness and size, and that is the type that will be returned.  */
